### PR TITLE
upgrade version to 1.6.0

### PR DESCRIPTION
**Pull request by zhoney**
Change-Id: Ic81ff6617ff962b015c7b25149a5c05dc878939e

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      - name: Prepare env and service
+        run: |
+          $TRAVIS_DIR/install-hugegraph-from-source.sh $COMMIT_ID
+      - name: Run test
+        run: |
+          mvn test -Dtest=FuncTestSuite
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          file: target/jacoco.xml
```
Comment (javeme) [inline .github/workflows/ci.yml]: blank line

File: pom.xml [hunk 2]
```diff
@@ -20,7 +20,7 @@
         <java.version>1.8</java.version>
         <source.encoding>UTF-8</source.encoding>
         <maven.build.timestamp.format>yyyy-MM-dd HH:mm:ssZ</maven.build.timestamp.format>
-        <hugegraph-client-version>1.9.3</hugegraph-client-version>
+        <hugegraph-client-version>2.0.0</hugegraph-client-version>
```
Comment (javeme) [inline pom.xml]: the final version is expected to be 2.0.1

File: assembly/travis/install-hugegraph.sh [hunk 3]
```diff
@@ -1,45 +1,46 @@
 #!/bin/bash
-
 set -ev
-
-TRAVIS_DIR=`dirname $0`
-
-if [ $# -ne 1 ]; then
-    echo "Must pass base branch name of pull request"
+if [[ $# -ne 1 ]]; then
```
Comment (javeme) [inline assembly/travis/install-hugegraph.sh]: also update conf if needed

File: assembly/travis/conf/graphs/hugegraph.properties [hunk 4]
```diff
@@ -42,7 +42,7 @@ cassandra.password=
 # hbase backend config
 #hbase.hosts=localhost
 #hbase.port=2181
-#hbase.znode_parent=/hbase
+#hbase.znode_parent=/hbsae
```
Comment (javeme) [inline assembly/travis/conf/graphs/hugegraph.properties]: hbase?

File: assembly/travis/install-hugegraph-from-source.sh [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+cd ../
+rm -rf "${GIT_DIR}"
+
+HTTP_SERVER_DIR=$(echo hugegraph-*)
+HTTPS_SERVER_DIR="hugegraph_https"
+
+cp -r "${HTTP_SERVER_DIR}" "${HTTPS_SERVER_DIR}"
+
+# config auth options just for http server (must keep '/.')
+cp -rf "${TRAVIS_DIR}"/conf/. "${HTTP_SERVER_DIR}"/conf/
```
Comment (javeme) [inline assembly/travis/install-hugegraph-from-source.sh:29]: need to copy TRAVIS_DIR files from client project to tools project

**Review by javeme — DISMISSED** [review]

**Review by javeme — APPROVED** [review]

**Review by imbajin — APPROVED** [review]

**Pull request merged**